spec/db/chores/proposal_cleaner_spec.rb

```diff
@@ -5,20 +5,22 @@
     it "removes proposals without client data" do
       _client_data_with_proposal = create(:ncr_work_order)
       create(:proposal, client_data: nil)
+      orig_count = Proposal.count
 
-      expect {
-        ProposalCleaner.new.run
-      }.to change { Proposal.count }.from(2).to(1)
+      ProposalCleaner.new.run
+
+      expect(Proposal.count).to eq(orig_count - 1)
```

Comment:
Heh! I didn't realize that we could do `.by(-1)` to check for a decrease. I'll give that a shot.